bin/civi-test-run

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 set -e
-PHPUNIT=phpunit5
+
+function getPhpunitVer() {
+  CIVIVER=$(getCiviVer)
+  if [ $(php -r "echo version_compare('$CIVIVER', '5.21', '>=');") ]; then
```

Comment:
On the good - on a local build with 5.22, that updated command was generally running :+1:

To check this detail (`5.21`), I did a little spot-checking - from https://github.com/civicrm/civicrm-wordpress/pull/175/commits , determine the commit id 541cfd457de9032f8f1ed568fb1863f6417ddf93 and check which branches have the commit.

```
[bknix-max:~/bknix/build/wpmaster/web/wp-content/plugins/civicrm] git branch --contains 541cfd457de9032f8f1ed568fb1863f6417ddf93
  5.22
  master
```

So doesn't this mean it should be >= 5.22?

Reply:
ah yeh oops